test/ml/test_ml.py

```diff
@@ -18,25 +18,32 @@ def setup_class(cls):
         Path(OUT_DIR).mkdir(parents=True, exist_ok=True)
 
     def test_summarize_networks(self):
-        dataframe = ml.summarize_networks([INPUT_DIR + 's1/s1.txt', INPUT_DIR + 's2/s2.txt', INPUT_DIR + 's3/s3.txt',
-                                           INPUT_DIR + 'longName/longName.txt', INPUT_DIR + 'longName2/longName2.txt',
-                                           INPUT_DIR + 'empty/empty.txt'])
+        dataframe = ml.summarize_networks([INPUT_DIR + 'test-s1/s1.txt', INPUT_DIR + 'test-s2/s2.txt', INPUT_DIR + 'test-s3/s3.txt',
+                                           INPUT_DIR + 'test-longName/longName.txt', INPUT_DIR + 'test-longName2/test-longName2.txt',
```

Comment:
We will investigate why this code ran with the `test-longName2/test-longName2.txt` file that doesn't exist

Reply:
We saw that this does fail with missing files here https://github.com/Reed-CompBio/spras/actions/runs/4876715224/jobs/8700546913#step:6:423